packages/web-sdk/src/instrumentations/performance/navigation.ts

```diff
@@ -1,12 +1,15 @@
 import { genShortID } from '@grafana/faro-core';
 import type { EventsAPI } from '@grafana/faro-core';
+import type {PushEventOptions} from "@grafana/faro-core/src/api/events/types";
```

Comment:
As per comment above, please add it to the main index.ts file so we can import it form `'@grafana/faro-core'`

Reply:
Exported